src/main/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporter.java

```diff
@@ -269,8 +271,18 @@ public void export(final OutputStream stream) throws IOException {
     private PushGateway buildPushGateWay() {
         // host:port or ip:port of the Pushgateway.
         String host = System.getenv("PROMETHEUS_PUSHGATEWAY_ADDRESS");
-        if(host != null){
-            return new PushGateway(host);
+        if (host != null) {
+            if (host.toLowerCase().contains("http://") || host.toLowerCase().contains("https://")) {
+                try {
+                    // allow to defne https urls as prometheus pushgateway address
+                    return new PushGateway(new URL(host));
```

Comment:
Do we even need the if here or can this constructor also be used for `http` URLs?